docs/package-lock.json

```diff
@@ -1,10216 +0,0 @@
-{
```

Comment:
I need to regenerate this with public URLs.

Reply:
🤦 i saw this after leaving my other comment